MYREADME.md

```diff
@@ -1,7 +1,7 @@
 # News Summary challenge
 
 ## How my Site looks
-[[https://i.imgur.com/kP8Z4iX.jpg]](https://imgur.com/kP8Z4iX)
+![Main_Site](https://i.imgur.com/kP8Z4iX.jpg)
```

Comment:
Wonderful to see you adding this. I did not add a screenshot this time, but will do it later. Well done Chris!